tools/fleetbot/webhook-handler.js

```
@@ -0,0 +1,372 @@
+      await github.addReviewCommentReaction(commentId, "eyes");
+    }
+    console.log(`[webhook] Added 👀 reaction to comment ${commentId}`);
+  } catch (err) {
+    console.warn(`[webhook] Failed to add reaction: ${err.message}`);
+  }
+
+  // Fetch the files changed in the PR
+  console.log(`[webhook] Fetching changed files for PR #${prNumber}...`);
+  const prFiles = await github.getPullRequestFiles(prNumber);
+  const activeFiles = prFiles.filter((f) => f.status !== "removed");
+  console.log(`[webhook] ${activeFiles.length} files to read from branch ${pr.headBranch}`);
```

ESLint

[error] Replace ``[webhook]·${activeFiles.length}·files·to·read·from·branch·${pr.headBranch}`` with `⏎····`[webhook]·${activeFiles.length}·files·to·read·from·branch·${pr.headBranch}`⏎··`

(prettier/prettier)